doc/articles/migrating-from-previous-releases.md

```diff
@@ -6,6 +6,42 @@ uid: Uno.Development.MigratingFromPreviousReleases
 
 This article details the migration steps required to migrate from one version to the next when breaking changes are being introduced.
 
+## Uno Platform 5.3
+
+Uno Platform 5.3 contains an improved template and Uno.SDK versioning, new default text font and Rider support.
+
+The support for .NET 7 has been removed, since it was a Standard Term Support (STS) release which [ended in May 2024](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core). You can [upgrade your project to .NET 8](xref:Uno.Development.MigratingFromNet7ToNet8) using our guide.
+
+### Single Project updates
```

Comment:
@jeromelaban no additional updates needed whatsoever for the https://github.com/unoplatform/uno/blob/master/doc/articles/features/using-the-uno-sdk.md documentation?

Also for the supported OS version, it is still the same lately I imagine?
https://github.com/unoplatform/uno/blob/master/doc/articles/features/using-the-uno-sdk.md#supported-os-platform-versions

Reply:
There's no other upgrade needed in using-the-uno-sdk.md, and supported versions did not change. They may when net9 gets released, but not yet.